specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```
@@ -0,0 +1,383 @@
+        ],
+        "description": "List all security contact configurations for the subscription",
+        "operationId": "SecurityContacts_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContactList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/securityContacts/{securityContactName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get a security contact": {
+            "$ref": "./examples/SecurityContacts/GetSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Get Default Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContact"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create security contact data": {
+            "$ref": "./examples/SecurityContacts/CreateSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Create security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Create",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "201": {
```

200?
201 if the resource was created and was not there before

```suggestion
          "200": {
```